src/element/resizeElements.ts

```diff
@@ -418,27 +418,27 @@ const resizeSingleElement = (
     element.width,
     element.height,
   );
-  const boundsOnScreenWidth = esx2 - esx1;
-  const boundsOnScreenHeight = esy2 - esy1;
+  const boundsCurrWidth = esx2 - esx1;
+  const boundsCurrHeight = esy2 - esy1;
```

Comment:
`Curr` for `Current`? Let's have the whole word

Reply:
Done